packages/0xcert-ethereum-generic-provider/src/core/mutation.ts

```diff
@@ -239,6 +239,77 @@ export class Mutation extends EventEmitter implements MutationBase {
     return this.resolveCurrentState();
   }
 
+  /**
+   * Retries mutation with higher gas price if possible. It uses providers retryGasMultiplier to
```

Comment:
* Retries mutation with a higher gas price if possible. It uses the provider's retryGasMultiplier to calculate the gas price.